pkg/cli/admin/admin.go

```diff
@@ -3,6 +3,8 @@ package admin
 import (
 	"fmt"
 
+	"github.com/openshift/oc/pkg/cli/admin/compare"
```

Comment:
Could be cleaner to move this together with the "github.com/openshift/oc/pkg/cli/admin/*" imports group in lines 17-41